tests/customer-resource-show-test.js

```diff
@@ -111,17 +111,21 @@ describeApplication('CustomerResourceShow', () => {
       expect(ResourcePage.isSelected).to.equal(false);
     });
 
+    it.still('should not display the back button', () => {
+      expect(ResourcePage.$backButton).to.not.exist;
+    });
+
     describe('successfully selecting a package title to add to my holdings', () => {
       beforeEach(function () {
         /*
-         * The expectations in the convergent `it` blocks
-         * get run once every 10ms.  We were seeing test flakiness
-         * when a toggle action dispatched and resolved before an
-         * expectation had the chance to run.  We sidestep this by
-         * temporarily increasing the mirage server's response time
-         * to 50ms.
-         * TODO: control timing directly with Mirage
-         */
+        * The expectations in the convergent `it` blocks
+        * get run once every 10ms.  We were seeing test flakiness
+        * when a toggle action dispatched and resolved before an
+        * expectation had the chance to run.  We sidestep this by
+        * temporarily increasing the mirage server's response time
+        * to 50ms.
+        * TODO: control timing directly with Mirage
+        */
```

Comment:
Looks like your editor accidentally got to this whitespace. Maybe it's because the comment doesn't start with `/**` like other doc-blocks?